lib/chiketto/organizer.rb

```diff
@@ -5,12 +5,8 @@ class Organizer < Resource
     attr_attrib :description
 
     def self.create(params)
-      response = Organizer.post 'organizer_new', params
-      if response.fetch('process', false)
-        response['process']['id']
-      elsif response.fetch('error', false)
-        raise response['error']['error_message']
-      end
+      response = Organizer.post 'organizers', params
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.